modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -1074,6 +1074,22 @@ func (h *HTTPTransport) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 			}
 			h.ForwardProxyURL = d.Val()
 
+		case "forward_proxy":
+			if !d.NextArg() {
+				return d.ArgErr()
+			}
+			modStem := d.Val()
+			modID := "caddy.network_proxy.source." + modStem
+			unm, err := caddyfile.UnmarshalModule(d, modID)
+			if err != nil {
+				return err
+			}
+			// TODO: should we validate here?
```

Comment:
I don't think so, it's fine for provision to do it

Reply:
Removed